lib/task_helpers/development/replication.rb

```diff
@@ -119,11 +119,11 @@ def run_command(command, raise_on_error: true, env: {})
         end
 
         def teardown_global_subscription_for_region(region)
-          run_command("psql -U #{PG_USER} #{database(GLOBAL)} -c 'drop subscription region_#{region}_subscription;'", raise_on_error: false)
+          run_command("psql -U #{PG_USER} #{database(GLOBAL)} -c 'DROP SUBSCRIPTION region_#{region}_subscription;'", raise_on_error: false)
         end
 
         def teardown_remote_publication(region)
-          run_command("psql -U #{PG_USER} #{database(region)} -c 'drop publication miq;'", raise_on_error: false)
+          run_command("psql -U #{PG_USER} #{database(region)} -c 'DROP PUBLICATION miq;'", raise_on_error: false)
```

Comment:
![for-jason](https://user-images.githubusercontent.com/19339/130685000-c34158f6-6c7c-4b12-8a42-d5760cfb6961.jpg)

Reply:
AND MY AXE! 🪓